src/lib/util/JCSSUtil.php

```diff
@@ -675,19 +688,24 @@ private static function readfile(&$contents, $file, $ext)
                                     $start = strpos($lineParseRest, '(') + 1;
                                     $end = strpos($lineParseRest, ')');
                                     $url = substr($lineParseRest, $start, $end - $start);
-                                    if ($url{0} == '"' | $url{0} == "'") {
-                                        $url = substr($url, 1, strlen($url) - 2);
-                                    }
-                                    // fix url
-                                    $url = dirname($file) . '/' . $url;
-                                    if (!$wasCommentHack) {
-                                        // clear buffer
-                                        $contents[] = $newLine;
-                                        $newLine = "";
-                                        // process include
-                                        self::readfile($contents, $url, $ext);
+                                    
+                                    if (!is_readable($url)) {
```

Comment:
Does this check for whether it is a remote url? Can this be done in a cheaper way avoiding file system access?